### PR TITLE
pipetools.get will return the results and better handle additional paramters

### PR DESCRIPTION
**Pull request by flokru**

### CODE REVIEW

**Review by jeskomerkel — COMMENTED** [review]
Probably up to @ccauet to review this...

**Review by jeskomerkel — APPROVED** [review]
Fetch single deal functionality works with the changes. For other features, you need approval from @ccauet

**Comment by flokru** [comment]
In order for our internal tools to work, some changes to Pipetools should be made. Is it possible to approve this PR?

**Review by ccauet — APPROVED** [review]

**Pull request merged**
